docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,120 @@
+# Etcd-Druid Local Setup
+
+This page aims to provide steps on how to setup Etcd-Druid locally with and without storage providers
+
+## Setting up etcd-druid
+
+### Clone the Etcd-Druid Github repo
+
+```sh
+# clone the repo
+git clone https://github.com/gardener/etcd-druid.git
+# cd into etcd-druid folder
+cd etcd-druid
+```
+
+---
+
+> **Note:** To setup Etcd-Druid with backups enabled on a [LocalStack](https://github.com/localstack/localstack) provider, refer [this document](https://github.com/gardener/etcd-druid/blob/master/docs/development/getting-started-locally-localstack.md)
+
+---
+
+### Setting up the Kind Cluster
+
+```sh
+# Create a kind cluster
+make kind-up
+```
+
+This creates a new kind cluster and stores the kubeconfig in the  `./hack/e2e-test/infrastructure/kind/kubeconfig` file.
+
+To target this newly created cluster, set the `KUBECONFIG` environment variable to the kubeconfig file located at `./hack/e2e-test/infrastructure/kind/kubeconfig` by using the following
+
+```sh
+export KUBECONFIG=$PWD/hack/e2e-test/infrastructure/kind/kubeconfig
+```
+
+### Setting up Etcd-Druid
+
+Generates the Etcd CRD and deploy an etcd-druid pod into the cluster
+
+```sh
+make deploy
+```
+
+### Applying the etcd CR
```

Comment:
Add a small point after this title before the 2 points
The note would say that we can create Etcd clusters in 2 flavours, the differences between them, and that they should use one of the two based on their use case